analyzer/testdata/src/comments/target_foo.go

```diff
@@ -0,0 +1,3 @@
+package comments
+
+//go:embed data.txt // want `don't use go:embed in _foo files`
```

Comment:
Hm, embed, should we bump go version in `go.mod` to 1.16 ?

Reply:
It's a testdata. I don't think it should be relevant. :)